index.html

```diff
@@ -47,24 +49,29 @@
         <article class="question-card">
           <p class="question-card__title">Question:</p>
           <p class="question-card__question">
-            Hier steht eine Frage:<br />
+            Hier steht eine Frage<br />
             Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore hic
             possimus eaque nam in perspiciatis laudantium doloribus enim.
           </p>
-          <button class="question-card__button">Show Answer</button>
-          <!-- /Hide Answer -->
-          <p class="question-card__answer">
-            Hier steht eine Antwort:<br />
+          <button class="question-card__button" data-js="button">
+            Show Answer
+          </button>
+          <p class="question-card__answer" data-js="answer">
+            Hier steht eine Antwort<br />
             Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore hic
             possimus eaque nam in perspiciatis laudantium doloribus enim.
           </p>
-          <ul class="question-card__tag--all">
-            <li class="question-card__tag">A</li>
-            <li class="question-card__tag">B</li>
-            <li class="question-card__tag">C</li>
-            <li class="question-card__tag">D</li>
+          <ul class="question-card__tag--all tag--all">
+            <li class="question-card__tag tag">A</li>
+            <li class="question-card__tag tag">B</li>
+            <li class="question-card__tag tag">C</li>
+            <li class="question-card__tag tag">D</li>
```

Comment:
I'm not sure if I understand why you decided to add new classes. 🤔  If I saw this correctly you already have the classes `question-card__tag` and `question-card__tag--all` in your file `home.css` (side note: Maybe a better name for this file is `card`, because this file contains all classes for your question cards.) that are currently empty, but could contain the stylings that you declared in your classes `tag--all` and `tag`. 
Instead of using `--` for your class name (as this means a modifier according to BEM naming convention) I recommend to use names like `question-cards__tags` and `question-card__tags__item`. 
I think you could also go with a shorter version here and just use `tags` and `tags__item`. Check this [BEM cheat sheet](https://9elements.com/bem-cheat-sheet/#list) that could be helpful for you and [the naming convention explanation for BEM (we use the two dashes style)](https://en.bem.info/methodology/naming-convention/#two-dashes-style). If you reuse the classes in another part of your app, you should definitely go with the shorter names and you could create a `tags.css` for these classes.